src/plugins/data_source_management/common/data_sources/data_sources/data_sources.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  OnError,
+  OnNotification,
+  SavedObject,
+  SavedObjectsClientCommon,
+  UiSettingsCommon,
+} from '../types';
+
+export interface DataSourceSavedObjectAttrs {
+  title: string;
+}
+
+interface DataSourcesServiceDeps {
+  uiSettings: UiSettingsCommon;
+  savedObjectsClient: SavedObjectsClientCommon;
+  onNotification: OnNotification;
+  onError: OnError;
+}
+
+export class DataSroucesService {
```

Comment:
typo

Reply:
Also I'd suggest a better name for class, cuz in the backend plugin, dataSource plugin, we'll also have `DataSourceService`

Reply:
We will remove this one here so no worries 😉

Reply:
removed this file